docs/controlling-fixtures/using-the-select-buttons-and-wheels.md

```diff
@@ -173,6 +192,15 @@ The options in the "Clear Options" submenu are:
 -   \[Clear/Maintain Cue Times\] - determines whether or not to clear
     the cue time information from the programmer (this will not affect
     fixture attribute times in the programmer).
+	
+-   \[Clear/Maintain Rate Settings\] - sets whether rate settings will be cleared from the programmer.	
+	
+-   \[Clear Direction\] - sets whether direction will be cleared from the programmer.	
+	
+-   \[Action Precedence\] - Sets the action for multiple presses of the \<Clear\> button:
+	- Selection With Programmer: **(default)** First press of the button clears both the fixture selection and all fixture changes in the programmer
+	- Selection Then Programmer: First press clears fixture selection, second press clears programmer
```

Comment:
I don't know whether I'm overthinking it, but it might be worth clarifying that the double-press functionality isn't related to how quickly you tap the button - it's related to the state of the selection (only). That is, with `Selection then Programmer`, the selection is cleared if it exists, otherwise, the programmer is cleared. With `Programmer then Selection`, the programmer is cleared if it isn't empty, otherwise, the selection is cleared.